pems_streamlit/src/pems_streamlit/components/plot_5_min_traffic_data.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+import plotly.graph_objs as go
+
+QUANTITY_CONFIG = {
+    "VOLUME_SUM": {"name": "Volume (veh/hr)"},
+    "OCCUPANCY_AVG": {"name": "Occupancy (%)"},
+    "SPEED_FIVE_MINS": {"name": "Speed (mph)"},
+}
+
+
+def plot_5_min_traffic_data(df_station_data: pd.DataFrame, quantities: list, lanes: list):
+    fig = go.Figure()
+
+    layout_updates = {
+        "xaxis": dict(title="Time of Day"),
+        "legend": dict(orientation="h", yanchor="top", y=-0.3, xanchor="center", x=0.5),
+    }
+
+    # One quantity selected
+    if len(quantities) == 1:
+        qty_key = quantities[0]
+        qty_name = QUANTITY_CONFIG[qty_key]["name"]
+
+        for lane in lanes:
+            df_lane = df_station_data[df_station_data["LANE"] == lane]
+            fig.add_trace(
+                go.Scatter(
+                    x=df_lane["SAMPLE_TIMESTAMP"],
+                    y=df_lane[qty_key],
+                    mode="lines",
+                    name=f"Lane {lane} {qty_name.split(' ')[0]}",
+                )
+            )
+
+        layout_updates["title"] = dict(text=f"<b>{qty_name}</b>", x=0.5, xanchor="center")
+        layout_updates["yaxis"] = dict(title=f"<b>{qty_name}</b>", side="right")
```

Comment:
I think for a single quantity, it is more standard to have the Y-axis on the `left` of the graph, right?

<img width="964" height="597" alt="Image" src="https://github.com/user-attachments/assets/ea03eab7-ca66-45c8-8775-2411fb089e94" />

Reply:
Ah thanks for catching that! For sure, I made the small change in bcbaf98045efd7391f5458d513c45bf8f07145da